lighthouse-core/report/html/report-styles.css

```diff
@@ -646,14 +648,25 @@
   border-bottom: 1px solid var(--report-secondary-border-color);
 }
 
+/*
+* Apply border-top to just the first audit.
+*/
+.lh-audit {
+  border-top: 1px solid var(--report-secondary-border-color);
+}
+.lh-audit ~ .lh-audit {
```

Comment:
this is a somewhat expensive selector. can we just do border-top on the `.lh-audit:nth-child(1)` ?

Reply:
done. had to be 2.

Reply:
ah. i see. i was looking at best practices where the DOM is a little different.

![image](https://user-images.githubusercontent.com/39191/56758967-ada5a400-674c-11e9-9479-862a540e8d3b.png)

hmm. if we can still keep to something simple like nth-child and support both these cases, i'd like to. but if its a PITA then perhaps we have to go back to the tilde.

Reply:
I'd prefer the tilde. this dom changes too much, and this would easily get out of date